concepts/regular-expressions/introduction.md

```diff
@@ -0,0 +1,188 @@
+# Introduction
+
+## 1. Regular Expressions in JavaScript
+
+A [Regular Expression][mdn-regular-expressions] (or Regex) is a sequence of characters that we can use to target and manipulate certain elements in strings. Hence, we can:
+
+- **Search** for a text in a string
+- **Replace** substrings in a string
+- **Extract** information from a string
+
+> 💡 JavaScript's regex flavor is part of the ECMA-262 standard for the language. This means that you don't have to worry about browser-specific compatibility.
+
+## 2. How to create Regular Expressions
+
+In JavaScript, a regular expressions is mostly written in the format `/pattern/modifiers` where 'pattern' is the main regex, and 'modifiers' are a series of characters which we can add to indicate special options (like superpowers).
+
+Nevertheless, we have two ways of creating a regular expression:
+
+1. **Regular Expression Literal**:
+
+```javascript
+const regex = /[a-z]/;
+```
+
+2. Constructor **RegExp**:
+
+```javascript
+const regex = new RegExp('[a-z]');
+```
+
+In both methods, JavaScript is also creating an Object out of the regex. It is recommended to use immutable patterns with the literal, and the RegExp for those regex which are probably going to change or we don't know yet (like an input).
```

Comment:
Thanks for the wording improvement! As English not being my mother language, I appreciate these corrections.

I'm seeing a button for committing the suggestions directly from this comment, nevertheless I'll just change it manually (Just in case I do something wrong).